packages/manager/apps/web/client/app/private-database/state/private-database-state.html

```diff
@@ -320,10 +320,10 @@
                         >
                             <oui-tile-description>
                                 <a
-                                    data-ng-href="{{:: stateCtrl.userLink }}"
+                                    data-ng-href="{{:: stateCtrl.userLink}}"
                                     class="oui-link"
                                     data-translate="privateDatabase_dashboard_admin_informations_users"
-                                    target="_top"
+                                    target="_self"
```

Comment:
```suggestion

```
_self is default target, so i think no need. You can remove target

Reply:
no issues, it's an explicit declaration.